webapp/components/login/login_controller.jsx

```diff
@@ -387,10 +390,16 @@ export default class LoginController extends React.Component {
                                 type='submit'
                                 className='btn btn-primary'
                             >
-                                <FormattedMessage
+                                {this.state.loading ? <span>
+                                    <span className='fa fa-refresh icon--rotate'/>
+                                    <FormattedMessage
+                                        id='login.signInLoading'
+                                        defaultMessage='Signing in...'
+                                    />
+                                </span> : <FormattedMessage
```

Comment:
@jwilander I made the changes. As it is in a push function, I wasn't sure if I should add the test outside or place it inside that's why I finally opted for the ternary.